source/deploy/mattermost-changelog.md

```diff
@@ -0,0 +1,81 @@
+# Mattermost changelog
+
+[Mattermost](https://mattermost.com) is an open source platform for secure collaboration across the entire software development lifecycle. 
+
+```{Important}
+From Mattermost v9.2, this changelog summarizes updates for the latest cloud and self-hosted versions of Mattermost to be [deployed and upgraded on infrastructure you control](https://docs.mattermost.com/guides/deployment.html).
+
+- See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) documentation for details on upgrading.
+- See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
+- **Self-Hosted Releases Prior to v9.2**: See the [Mattermost Legacy Self-Hosted Changelog](/deploy/legacy-self-hosted-changelog) for details.
+- **Cloud Releases Prior to v9.2**: See the [Mattermost Legacy Cloud Changelog](/deploy/legacy-cloud-changelog) for details.
+```
+
+```{contents} On ths page
+:depth: 2
+```
+
+## Release v9.2 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
```

Comment:
Added v9.2 changelog content below from https://github.com/mattermost/docs/pull/6717/files. To be added: list of contributors; potential changes to the list of bug fixes and known issues.